### PR TITLE
Value of trace_configs might be None

### PR DESCRIPTION
**Pull request by estheruary**
# Description

Ran into this in the wild, `trace_configs` might be None.

```
future: <Task finished name='Task-1' coro=<Client.start() done, defined at /Users/---/.pyenv/versions/---/lib/python3.11/site-packages/discord/client.py:724> exception=TypeError("'NoneType' object is not iterable")>
Traceback (most recent call last):
  File "/Users/---/.pyenv/versions/---/lib/python3.11/site-packages/discord/client.py", line 745, in start
    await self.login(token)
  File "/Users/---/.pyenv/versions/---/lib/python3.11/site-packages/discord/client.py", line 580, in login
    data = await self.http.static_login(token)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/---/.pyenv/versions/---/lib/python3.11/site-packages/discord/http.py", line 789, in static_login
    self.__session = aiohttp.ClientSession(
                     ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/---/.pyenv/versions/---/lib/python3.11/site-packages/opentelemetry/instrumentation/aiohttp_client/__init__.py", l
ine 283, in instrumented_init
    client_trace_configs = list(kwargs.get("trace_configs", ()))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not iterable
```

```
trace_configs is Optional[Sequence]

None -> kwargs.get("trace_configs", ()) == None --> TypeError
```

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [x] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py [hunk 1]
```diff
@@ -280,7 +280,7 @@ def instrumented_init(wrapped, instance, args, kwargs):
         if context_api.get_value(_SUPPRESS_INSTRUMENTATION_KEY):
             return wrapped(*args, **kwargs)
 
-        client_trace_configs = list(kwargs.get("trace_configs", ()))
+        client_trace_configs = list(kwargs["trace_configs"] or [])
```
Comment (shalevr) [inline instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py:283]: If trace_configs did not exist in kwargs the code throws a KeyError exception, so you need to verify this and use kwargs.get().

client_trace_configs = list(kwargs.get("trace_configs") or [])

Comment (estheruary) [reply instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py]: Done!

**Review by shalevr — APPROVED** [review]

**Review by srikanthccv — APPROVED** [review]

**Comment by srikanthccv** [comment]
Please add CHANGELOG entry

**Comment by estheruary** [comment]
> Please add CHANGELOG entry

Done!

**Comment by nemoshlag** [comment]
Thanks @estheruary for fixing my bug 🙏

**Pull request merged**